src/UartSerial.cpp

```diff
@@ -156,7 +157,7 @@ UartSerial::begin (
     // Leave all output flags unset
 
     // c_cflag - control modes
-    _tio_config.c_cflag |= CS8;  //TODO: Configuration passed by caller
+    _tio_config.c_cflag |= c_cflags;
```

Comment:
So I thought about that but we know exactly which bits we want set so I didn't see the value in keeping the original config and then turning off the things we don't want. As is we just start from scratch with nothing turned on and then only turn on those things necessary for the connection to work. I feel like there could be a potential for errors if anything new is introduced in the termios struct that we'd want to turn off. Then we'd need to come in and explicitly turn it off to make our code work again. Thoughts?